File: taskmanager/src/main/java/com/example/alexeyglushkov/taskmanager/loader/http/HttpURLConnectionProvider.java
package com.example.alexeyglushkov.taskmanager.loader.http;

import java.net.HttpURLConnection;
import java.net.URL;

/**
 * Created by alexeyglushkov on 02.08.15.
 */

public interface HttpURLConnectionProvider {
    HttpURLConnection getUrlConnection();
    URL getURL();
}

File: taskmanager_httptask/src/main/java/com/aglushkov/taskmanager_http/loader/http/HttpURLConnectionProvider.kt
package com.aglushkov.taskmanager_http.loader.http

import java.net.HttpURLConnection
import java.net.URL

/**
 * Created by alexeyglushkov on 02.08.15.
 */
interface HttpURLConnectionProvider {
    val url: URL
    fun getUrlConnection(): HttpURLConnection
}